.github/workflows/test.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           go-version-file: go.mod
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@25e2cdc5eb1d7a04fdc45ff538f1a00e960ae128 # v8.0.0
```

Comment:
Weird that the hash has changed, but not the version. Potentially indicative of a supply chain attack? We should investigate before merging